### ISSUE TITLE
No documentation on sharing projects so others can work on them.

### ISSUE THREAD

**Issue by JohnGalt1717** [MicrosoftTranslator/Custom-Translator-Documentation#1]
How does one do this? I can't find any way to have others in my organization have access to collaborate and the projects are locked to my account which is bad too. I'm trying to correct that but I see nothing in the documentation nor the UI.

**Comment by Taroka** [comment]
I have the same issue. I want to know how to connect with Translator Speech API
https://docs.microsoft.com/en-us/azure/cognitive-services/translator-speech/quickstarts/python

Also, I found two similar speech translation API under cognitive service webpage. However, I cannot find two different API in Azure portal..
https://azure.microsoft.com/en-us/services/cognitive-services/speech-translation/
https://azure.microsoft.com/en-us/services/cognitive-services/translator-speech-api/

**Comment by JohnGalt1717** [comment]
@Taroka The 2 are the old and the new version.  

But this new version of the translator tools really needs to be updated and hooked into your Office 365 org so that you can have multiple people working on this stuff. It's brutal right now.

Note that under settings you can specify the Translator Api key.

**Comment by Taroka** [comment]
@JohnGalt1717 I use my organization Office 365 account to log in to Azure Azure. However, I cannot use my org Office 365 to log in Custom Translation. 

:(

**Comment by JohnGalt1717** [comment]
@Taroka Yes, it's a mess and not seeing any responsiveness to get this fixed.

**Comment by kellyaltom** [comment]
Please send Custom Translator questions to: custommt@microsoft.com. This is the alias for the Custom Translator team.


________________________________
